internal/benchrunner/runners/system/metrics.go

```diff
@@ -193,12 +295,11 @@ func (c *collector) waitUntilReady() {
 
 readyLoop:
 	for {
-		if signal.SIGINT() {
-			logger.Debug("SIGINT: cancel metrics collection")
+		select {
+		case <-c.stopC:
```

Comment:
Is `stopC` also closed on SIGINT?

Reply:
Yes, since when SIGINT happens runner ends gracefully, and the collector `stop` function is always executed, either after collection or on early return on `runner.Run`